src/helpers/insight-matrix/value-threshold.ts

```
@@ -0,0 +1,48 @@
+            return undefined;
+        }
+        return NotEnteredThreshold;
+    };
+
+    let currentDate = startDate;
+    while (currentDate <= endDate) {
+        const currentDayKey = getDayKey(currentDate);
+        const threshold = computeThreshold(overviewData.configuration.thresholds, overviewData.result[currentDayKey]);
+
+        if (threshold) {
+            if (!lookup.hasOwnProperty(threshold)) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)